app/dic.ts

```diff
@@ -16,7 +16,24 @@ const container = awilix.createContainer({
 });
 
 export function configcontainer(envVariables: EnvVariables) {
-  return createConnection().then(connection => {
+  return createConnection({
```

Comment:
Database functionality could be moved to it's own file since it is getting pretty big and dic knows too much about it :D